### PR TITLE
Check subscription key on history page refresh

### PR DESCRIPTION
**Pull request by nizetic**

### CODE REVIEW

File: src/components/bread-crumb-nav/bread-crumb-nav.js [hunk 1]
```diff
@@ -1,42 +1,50 @@
...
 
-  const splitPaths = getSplitPaths(pathname);
+  const splitPaths = getSplitPaths(currentPath);
   const itemsWithHeading = [];
 
-  splitPaths.forEach(path => {
-    if (ROUTE_NAME_BY_PATH.hasOwnProperty(path)) {
+  useEffect(() => {
+    if (!subKey) {
+      navigate(PATHS.INDEX, { replace: true });
```
Comment (shnooshnoo) [inline src/components/bread-crumb-nav/bread-crumb-nav.js:26]: similar code exists here https://github.com/Azure/azure-maps-creator-onboarding-tool/blob/main/src/components/route/route.js#L31, i guess one of these can be removed

File: src/components/bread-crumb-nav/bread-crumb-nav.js [hunk 2]
```diff
@@ -1,42 +1,50 @@
...
+import { shallow } from 'zustand/shallow';
 import { breadcrumbStyle } from './bread-crumb-nav.style';
-import { getSplitPaths } from 'common/functions';
-import { PATHS, ROUTE_NAME_BY_PATH } from 'common/constants';
-import { useConversionStore } from 'common/store';
 
-const routesConfirmationRequiredBeforeLeaving = [PATHS.CONVERSION];
+const routesReset = [PATHS.CONVERSION];
+
+const userStoreSelector = s => [s.subscriptionKey];
```
Comment (shnooshnoo) [inline src/components/bread-crumb-nav/bread-crumb-nav.js:13]: minor thing: when selecting a single value it's better not to use arrays but return it directly s => s.subKey.
With this you won't need to add shallow as it is redundant with primitives

File: src/components/bread-crumb-nav/bread-crumb-nav.js (hunk 1 above)
Comment (nizetic) [reply src/components/bread-crumb-nav/bread-crumb-nav.js:26]: Fixed in https://github.com/Azure/azure-maps-creator-onboarding-tool/pull/104

File: src/components/bread-crumb-nav/bread-crumb-nav.js (hunk 2 above)
Comment (nizetic) [reply src/components/bread-crumb-nav/bread-crumb-nav.js:13]: Fixed in https://github.com/Azure/azure-maps-creator-onboarding-tool/pull/104

**Pull request merged**